sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/ViewCatalog.java

```diff
@@ -129,10 +129,11 @@ default boolean viewExists(Identifier ident) {
    *
    * @param viewInfo the info class holding all view information
    * @param orCreate create the view if it doesn't exist
+   * @return the created/replaced created
```

Comment:
shall we follow https://github.com/apache/spark/commit/d8fb91e61352e57e733e7d7e4978c8ce555454b1#diff-76ba8535886252d11f5de158f6e465ea92dc07f3e2b3f6a976a5cb412fb19112R190 ?

Reply:
Yea only change the doc for now. We will need to deal with null when implementing the v2 view commands.

Reply:
I've updated the doc for create + replace view